src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -95,7 +102,7 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
                     {/*
                     // @debt typing - need to support known User interface with unknown question keys
                     // @ts-ignore */}
-                    {userProfile[question.name] || //@debt typing - look at the changelog, was this a bug?
+                    {selectedUserProfile[question.name] || //@debt typing - look at the changelog, was this a bug?
                       "I haven't edited my profile to tell you yet"}
```

Comment:
This default text should possibly also be moved to a const in `src/settings.ts` and then referenced here.